tests/pl/_fixtures.yaml

```diff
@@ -22,6 +52,33 @@ entities:
   - name: Wentylator sufitowy
     id: fan.ceiling
     area: living_room
+  - name: Wentylator stołowy
+    id: fan.standalone
+    area: bedroom
+  - name: Wentylacj(a|ę)
+    id: fan.ventilation
+    area: wc
   - name: Rolet(a|ę)
     id: cover.roleta
     area: living_room
+  - name: Ekspers do kawy | Ekspres
```

Comment:
Typo - should be `Ekspres` not `Ekspers`

Reply:
Ohh, ups 😨 I've added it as a bug to our issue tracker and push a fix later today, thanks for noticing it!
https://github.com/Uriziel01/home-assistant-intents-pl/issues/6